### PR TITLE
Added ability to use tags in new item text: today, tomorrow, p1 to p4

### PR DESCRIPTION
**Pull request by ni-ka**
First attempt at addressing #367. Only in app so far, not in quick-add since it uses separate code.

### CODE REVIEW

File: src/Utils.vala [hunk 1]
```diff
@@ -1016,4 +1016,42 @@ public class Utils : GLib.Object {
...
+        Regex wordRegex = /\S+\s*/;
+        MatchInfo matchInfo;
+        
+        var matchText = text. strip ();
+        for (wordRegex.match (matchText, 0, out matchInfo) ; matchInfo.matches () ; matchInfo.next ()) {
+            var word = matchInfo.fetch (0);
+            var stripped =    word.strip ().down ();
+
+            switch (stripped) {
+                case "today":
```
Comment (alainm23) [inline src/Utils.vala]: Use `_("today")` to make the text translatable

File: src/Utils.vala [hunk 2]
```diff
@@ -1016,4 +1016,42 @@ public class Utils : GLib.Object {
...
+                    item.due_date = new GLib.DateTime.now_local ().to_string ();
+                    break;
+                case "tomorrow":
```
Comment (alainm23) [inline src/Utils.vala]: Use `_("tomorrow")` to make the text translatable

**Review by alainm23 — CHANGES_REQUESTED** [review]
Thank you very much, this is a great first step. I just have a few observations. Later I would like to make it more visual.

File: src/Utils.vala (hunk 1 above)
Comment (ni-ka) [reply src/Utils.vala]: will do!

File: src/Utils.vala (hunk 2 above)
Comment (ni-ka) [reply src/Utils.vala]: will do!

**Comment by ni-ka** [comment]
I've added the translation texts as constants as it would not let me use _() in the case statement.

> Later I would like to make it more visual.
Yeah, I've started playing around with replacing the entry text with a GtkSourceTextView to have completion (for keywords, projects and labels) and being able to add visual tags, but it'll probably take me a bit.

**Pull request merged**